src/AppBundle/Command/ConnectionFollowUpEmailCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\ArrayInput;
+use AppBundle\Entity\Job;
+use AppBundle\Entity\Connection;
+
+/**
+ * Class ConnectionFollowUpEmailCommand
+ * @package AppBundle\Command
+ */
+class ConnectionFollowUpEmailCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('kb:connection:followup')
+            ->setDescription('Follow up mail after connection has made')
+            ->addArgument(
+                'days',
+                InputArgument::OPTIONAL,
+                'Number of days ago',
+                Connection::FOLLOW_UP_14_DAYS
+            )
+        ;
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $days       = $input->getArgument('days');
+        $jobManager = $this->getContainer()->get('job_manager');
+        $jobType    = ($days == Connection::FOLLOW_UP_14_DAYS? Job::CONNECTION_MAIL_FOLLOW_UP_14: Job::CONNECTION_MAIL_FOLLOW_UP_42);
+
+        if ($jobManager->isRunning($jobType)) {
```

Comment:
Use the LockHandler instead http://symfony.com/doc/current/components/filesystem/lock_handler.html